code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -56,142 +56,189 @@
-				H.feral = 0
-				H << "<span class='info'>Your thoughts start clearing, your feral urges having passed - for the time being, at least.</span>"
-				log_and_message_admins("is no longer feral.", H)
-
-// handle what happens while feral
-
-	if(H.feral > 0) //do the following if feral, otherwise no effects.
-		var/light_amount = H.getlightlevel() //how much light there is in the place
-
-		H.shock_stage = max(H.shock_stage-(H.feral/20), 0) //if they lose enough health to hit softcrit, handle_shock() will keep resetting this. Otherwise, pissed off critters will lose shock faster than they gain it.
-
-		if(light_amount <= 0.5) // in the darkness. No need for custom scene-protection checks as it's just an occational infomessage.
-			if(prob(2)) //periodic nagmessages just to remind 'em they're still feral
-				if (H.traumatic_shock >=min(60, H.nutrition/10)) // if hurt, tell 'em to heal up
-					H << "<span class='info'> This place seems safe, secure, hidden, a place to lick your wounds and recover...</span>"
-				else if(H.nutrition <= 100) //If hungry, nag them to go and find someone or something to eat.
-					H << "<span class='info'> Secure in your hiding place, your hunger still gnaws at you. You need to catch some food...</span>"
-				else if(H.jitteriness >= 100)
-					H << "<span class='info'> sneakysneakyyesyesyescleverhidingfindthingsyessssss</span>"
-				else //otherwise, just tell them to keep hiding.
-					H << "<span class='info'> ...safe...</span>"
-		else // must be in a lit area
-			var/list/nearby = oviewers(H)
-			if (nearby.len) // someone's nearby
-				if(prob(1)) // 1 in 100 chance of doing something so as not to interrupt scenes
-					var/mob/M = pick(nearby)
-					if (H.traumatic_shock >=min(60, H.nutrition/10)) //tell 'em to be wary of a random person
-						H << "<span class='danger'> You're hurt, in danger, exposed, and [M] looks to be a little too close for comfort...</span>"
-					else if(H.nutrition <= 250 || H.jitteriness > 0) //tell them a random person in view looks like food. It CAN happen when you're not hungry enough to be feral, especially if coffee is involved.
-						H << "<span class='danger'> Every movement, every flick, every sight and sound has your full attention, your hunting instincts on high alert... In fact, [M] looks extremely appetizing...</span>"
-					if(H.stat == CONSCIOUS)
-						H.emote("twitch")
-					H.handle_feral()
-			else // nobody around
+// Handle being feral
+	if(feral)
+
+		//Shock due to mostly halloss. More feral.
+		if(2.5*H.halloss >= H.traumatic_shock)
+			H.feral = max(H.feral, H.halloss)
+
+		//Shock due to mostly injury. More feral.
+		else
+			H.feral = max(H.feral, H.traumatic_shock * 2)
+
+		//Still jittery? More feral.
+		if(H.jitteriness)
+			H.feral = max(H.feral, H.jitteriness-100)
+
+		//Still hungry? More feral.
+		if(H.feral + H.nutrition < 150)
+			H.feral++
+		else
+			H.feral = H.feral
+
+		//Did we just finish being feral?
```

Comment:
Can't see feral being decremented. "If (hungry || shock || jittery) do the above things, else feral--" maybe?

Reply:
Typo, it was intended to be on L167